server/datastore/mysql/labels.go

```diff
@@ -118,7 +118,7 @@ func batchHostnames(hostnames []string) [][]string {
 	// overflowing the MySQL max number of parameters (somewhere around 65,000
 	// but not well documented). Algorithm from
 	// https://github.com/golang/go/wiki/SliceTricks#batching-with-minimal-allocation
-	const batchSize = 50000 // Large, but well under the undocumented limit
+	const batchSize = 20_000 // Large, but well under the undocumented limit
```

Comment:
Let's add a WARNING comment that uses of `batchHostnames` must be checked before changing this value.